src/inc/FIRComb.h

```diff
@@ -0,0 +1,29 @@
+//
+//  IIRComb.hpp
+//  MUSI6106
+//
+//  Created by 李克镰 on 2/10/22.
+//
+
+#ifndef FIRComb_h
+#define FIRComb_h
+
+#include <stdio.h>
+#include <CombFilter.h>
+
+
+
+
+class CFIRComb: public::CCombFilterBase
```

Comment:
The extra colons here (between `public` and `CCombFilterBase`) made me do a double-take.
I think it compiles because it parses as `public ::CCombFilterBase` (where `::CCombFilterBase` just resolves to `CCombFilterBase` since it's already in the global namespace).
In any case, this could just be `class CFIRComb: public CCombFilterBase`.)